src/web/components/elements/GuVideoBlockComponent.tsx

```diff
@@ -13,18 +13,18 @@ export const GuVideoBlockComponent: React.FC<{
     credit: string;
     caption?: string;
 }> = ({ html, pillar, designType, display, credit, caption }) => {
+    const hasCaption = caption != null;
+
+    const embedContainer = css`
+        width: 100%;
```

Comment:
You lost `video { width: 100% }` here, checking that's correct?

Reply:
Nice catch